app-dicts/myspell-nn/myspell-nn-2.0.10.ebuild

```diff
@@ -20,7 +20,7 @@ MYSPELL_THES=(
 inherit myspell-r2
 
 DESCRIPTION="Norwegian dictionaries for myspell/hunspell"
-HOMEPAGE="http://spell-norwegian.alioth.debian.org/"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
```

Comment:
Is there no better alternative? Like the other norwegian myspell homepage, I'd say it's close enough?

Reply:
These are different languages followed by different project's so no, i can't just use same homepage

Reply:
Found it at the end, updating